tensorflow/python/compiler/tensorrt/test/BUILD

```diff
@@ -63,566 +59,54 @@ filegroup(
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "multi_connection_neighbor_engine_test",
-    srcs = [
-        "multi_connection_neighbor_engine_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "neighboring_engine_test",
-    srcs = [
-        "neighboring_engine_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "quantization_test",
-    srcs = [
-        "quantization_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "rank_two_test",
-    srcs = [
-        "rank_two_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "reshape_transpose_test",
-    srcs = [
-        "reshape_transpose_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "tf_function_test",
-    srcs = [
-        "tf_function_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
+oss_tests = [
```

Comment:
Why do you choose the name oss_tests? Would all_tests be better?

Reply:
To be honest, I thought  `all_test` should refer to the union of the disjoint sets of `oss_tests` and  `no_oss_test`.
The suffix `no_oss` was already used in 
```
test_suite(
    name = "tf_trt_integration_test_no_oss",
    ...
)
```
And I started to use the `oss_tests` name.